buildrunner/docker/runner.py

```diff
@@ -251,6 +255,94 @@ def cleanup(self):
 
         self.container = None
 
+    def restore_caches(self, caches: OrderedDict, cache_archive_ext: str):  # pylint: disable=too-many-branches,too-many-locals
+        """
+        Restores caches from the host system to the destination location in the docker container.
+        """
+        if caches is None or not isinstance(caches, OrderedDict):
+            print("WARNING: caches is not an OrderedDict")
```

Comment:
do we use print statements in buildrunner or should we be using some sort of logger ?